.circleci/config.yml

```diff
@@ -137,6 +145,23 @@ jobs:
       - run: ./scripts/circleci/run-tests.sh
       - *store_junit_test_results
 
+  # ------------------------------------------------------------------------------------------
+  # Jobs that snapshot the `master` branch and update the docs on commits to master
+  # ------------------------------------------------------------------------------------------
+  bower_material_snapshot:
+    <<: *job_defaults
+    steps:
+      - *checkout_code
+      - *restore_cache
```

Comment:
Missing npm install?

Reply:
The thought was that we don't need to do `npm ci` here since the cache includes `node_modules/` from the build that is required before this job.

Reply:
That's a fair point, but usually we just run `yarn`/`npm` again in order ensure that the modules are properly installed (these calls are a noop if the integrity is verified)

Also technically the cache is not always guaranteed to be stored as part of the same cache key. e.g. something changes the lock file in previous builds